### PR TITLE
Feature/7969 clear old environments

### PR DESCRIPTION
**Pull request by gavinwakeley1**
AB#7969
AB#7958

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -88,6 +102,8 @@ pool:
 jobs:
 - job: packageAndPublish
   displayName: Package And Publish Buyingcatalogue
+  # pool:
```
Comment (stuart-bass-bjss) [inline azure-pipelines.yml:105]: delete please

File: azure-pipelines.yml [hunk 2]
```diff
@@ -169,52 +187,14 @@ jobs:
   - bash: './determine-namespace.sh'
     name: DeployVars
     displayName: Set up environment variables for deployment
-
```
Comment (stuart-bass-bjss) [inline azure-pipelines.yml:172]: I don't think you should be removing all this?

File: housekeeping/azure-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+  - cron: "0 0 * * *"
+    displayName: Daily midnight Housekeeping Tasks
+    branches:
+      include:
+      - master
+      - feature/7969-clear-old-environments
+    always: true
+
+trigger: none
+#trigger:
```
Comment (stuart-bass-bjss) [inline housekeeping/azure-pipelines.yml:11]: why the commented out trigger?

File: housekeeping/azure-pipelines.yml [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+    - checkout: marketing-pages
+    - checkout: order-form
+    - checkout: OrderFormAcceptanceTests
+    - checkout: self
+      persistCredentials: true
+    - checkout: public-browse
+    - checkout: PublicBrowseAcceptanceTests
+
+#### Checks folder structure    
+#    - powershell: get-childitem -r  # inline script
```
Comment (stuart-bass-bjss) [inline housekeeping/azure-pipelines.yml:120]: remove unnecessary code?

File: housekeeping/azure-pipelines.yml [hunk 5]
```diff
@@ -0,0 +1,195 @@
+schedules:
+  - cron: "0 0 * * *"
+    displayName: Daily midnight Housekeeping Tasks
+    branches:
+      include:
+      - master
+      - feature/7969-clear-old-environments
```
Comment (stuart-bass-bjss) [inline housekeeping/azure-pipelines.yml:7]: can remove this branch name

**Comment by gavinwakeley1** [comment]
Issues with Pull Request and version comparison - closing down and will raise a new PR

File: housekeeping/azure-pipelines.yml (hunk 3 above)
Comment (gavinwakeley1) [reply housekeeping/azure-pipelines.yml:11]: Trigger not needed in housekeeping, it runs purely on the daily schedule

File: housekeeping/azure-pipelines.yml (hunk 5 above)
Comment (gavinwakeley1) [reply housekeeping/azure-pipelines.yml:7]: Removed branch from trigger

File: housekeeping/azure-pipelines.yml (hunk 4 above)
Comment (gavinwakeley1) [reply housekeeping/azure-pipelines.yml:120]: Removed

File: azure-pipelines.yml (hunk 2 above)
Comment (gavinwakeley1) [reply azure-pipelines.yml]: This was caused by merged changes into master after branch created. I have updated branch to reflect master

File: azure-pipelines.yml (hunk 1 above)
Comment (gavinwakeley1) [reply azure-pipelines.yml]: This was caused by merged changes into master after branch created. I have updated branch to reflect master

**Pull request merged**